rate_counter/templates/table_detail.html

```diff
@@ -0,0 +1,91 @@
+{% extends "base.html" %}
+
+{% block title %}
+
+    {{ table.name }} | {{ block.super }}
+
+{% endblock title %}
+
+{% block navbar %}
+
+    <li class="dropdown">
+        <a class="dropdown-toggle" data-toggle="dropdown" href="#">Table<span class="caret"></span></a>
+        <ul class="dropdown-menu">
+            <li><a href="{% url 'rate:delete_table' table.id %}">Delete</a></li>
+            <li><a href="{% url 'rate:edit_table' table.id %}">Rename</a></li>
+            <li><a href="{% url 'rate:tables_list' %}">See all</a></li>
+        </ul>
+    </li>
+
+{% endblock navbar %}
+
+{% block content %}
+
+            <h1>{{table.name}}</h1><br><br>
+            <form method="POST" action="{% url 'rate:edit_rows' table.id %}" enctype="multipart/form-data" >{% csrf_token %}
+                <table class="table-condensed">
+                    <thead>
+                        <tr>
+                            <th>Subject name</th>
+                            <th>Multiplier</th>
+                            <th>Grade</th>
+                        </tr>
+                    </thead>
+                    <tbody>
+                        {% for row in rows %}
+                        <tr>
+                            <td><input size="120" class="form-control input-md" onchange="submit" value="{{row.name}}" name="name{{row.id}}" type="text"  maxlength="100"></td>
+                            <td><input class="form-control input-md" id="row_num_input" onchange="submit" value="{{row.coefficient}}" name="coef{{row.id}}" type="number" step="0.5" min="0" max="10"></td>
```

Comment:
Here you are using same id selector in two same HTML elements. Id is for something identical :) here it is better to use  class. Also from my experience I am trying to avoid using id a lot.